packages/web-components/stencil.config.ts

```diff
@@ -7,6 +8,15 @@ export const config: Config = {
   namespace: 'component-library',
   globalStyle: 'src/global/main.css',
   plugins: [
+    sass({ 
+      includePaths: [
+        '../../node_modules/@uswds/uswds/packages/',
+        '../../node_modules/@department-of-veterans-affairs/css-library/src/tokens/',
```

Comment:
Is there any additional build steps that need to happen to get this to compile?

Reply:
No, that I can think of